internal/object/objects_test.go

```diff
@@ -0,0 +1,102 @@
+package object
+
+import (
+	"context"
+	"github.com/icinga/icinga-go-library/database"
+	"github.com/icinga/icinga-go-library/types"
+	"github.com/icinga/icinga-notifications/internal/event"
+	"github.com/icinga/icinga-notifications/internal/testutils"
+	"github.com/icinga/icinga-notifications/internal/utils"
+	"github.com/jmoiron/sqlx"
+	"github.com/stretchr/testify/assert"
+	"github.com/stretchr/testify/require"
+	"testing"
+	"time"
+)
+
+func TestRestoreMutedObjects(t *testing.T) {
+	ctx := context.Background()
+	db := testutils.GetTestDB(ctx, t)
+
+	var sourceID int64
+	err := utils.RunInTx(ctx, db, func(tx *sqlx.Tx) error {
+		args := map[string]interface{}{
+			"type":     "notifications",
+			"name":     "Icinga Notifications",
+			"insecure": "n",
+		}
+		// We can't use config.Source here unfortunately due to cyclic import error!
+		id, err := utils.InsertAndFetchId(ctx, tx, `INSERT INTO source (type, name, icinga2_insecure_tls) VALUES (:type, :name, :insecure)`, args)
```

Comment:
You shouldn't have to specify `icinga2_insecure_tls` here, that column has a default value.